FlowCrypt/Controllers/Setup/SetupEnterPassPhraseViewController.swift

```diff
@@ -151,30 +169,44 @@ extension EnterPassPhraseViewController: ASTableDelegate, ASTableDataSource {
                     return true
                 }
             case .enterPhrase:
-                return ButtonCellNode(
-                    title: self.decorator.passPhraseContine,
-                    insets: self.decorator.passPhraseInsets
-                ) { [weak self] in
+                let input = ButtonCellNode.Input(
+                    title: self.decorator.buttonTitle(for: .passPhraseContinue),
+                    insets: self.decorator.insets.buttonInsets
+                )
+                return ButtonCellNode(input: input) { [weak self] in
                     self?.handleContinueAction()
                 }
             case .chooseAnother:
-                return ButtonCellNode(
-                    title: self.decorator.passPhraseChooseAnother,
-                    insets: self.decorator.buttonInsets,
-                    color: .lightGray
-                ) { [weak self] in
+                return ButtonCellNode(input: .chooseAnotherAccount) { [weak self] in
                     self?.navigationController?.popViewController(animated: true)
                 }
             case .divider:
                 return DividerCellNode(inset: UIEdgeInsets(top: 0, left: 24, bottom: 0, right: 24))
+            case .saveLocally:
+                return self.saveLocallyNode
+            case .saveInMemory:
+                return self.saveInMemoryNode
             }
         }
     }
+
+    func tableNode(_ tableNode: ASTableNode, didSelectRowAt indexPath: IndexPath) {
+        guard let part = Parts(rawValue: indexPath.row) else { return }
+
+        switch part {
+        case .saveLocally:
+            shouldSaveLocally = true
+        case .saveInMemory:
+            shouldSaveLocally = false
+        default:
+            break
+        }
+    }
 }
 
 // MARK: - Actions
 
-extension EnterPassPhraseViewController {
+extension SetupEnterPassPhraseViewController {
```

Comment:
How is `SetupEnterPassPhraseViewController` different from `SetupBackupsViewController`? Both handle pass phrase input and both handle key import. They are written and named slightly differently but it's not clear what the difference is.

Maybe there is some overlapping functionality, or maybe some of the functionality in one of them is no longer in use?

Reply:
`SetupInitialViewController` is the first controller on flow. It's searchBackups for a backups and based on result if there are some backups found navigate to `SetupBackupsViewController`.
`SetupInitialViewController` contains a button which can navigate to `SetupImportKeyViewController` where in case user add keys (from pasteboard for example) you will be navigated to `SetupEnterPassPhraseViewController`

It's a way to separate responsibilities for a controllers. You can either have one huge view controller with all functionality in place, different states and so on or have few controllers. I decide to split them because it was much easier solution to maintain

Reply:
Got it. Are these flows shown correctly?

 - `SetupInitialViewController` -> `SetupBackupsViewController` (where I can also enter pass phrase) -> inbox
 - `SetupInitialViewController` -> `SetupManuallyImportKeyViewController` -> `SetupEnterPassPhraseViewController` -> inbox
 - `SetupInitialViewController` -> `SetupGenerateKeyViewController` (where I can also enter pass phrase) -> inbox

That means `SetupEnterPassPhraseViewController` is only used for importing keys. Is that correct? If yes, then let's name it in a way that suggests that they are related: `SetupManuallyEnterPassPhraseViewController`

Also, in another PR (I'm filing an issue), it would be good for each view controller to have a block comment above it where it's explained what is the view for and how user gets there and where user typically goes next. For example:

```ts
/**
 * user is sent here from SetupInitialViewController when some backups were found in his inbox
 * here the user can import their pass phrase that matches these backups to proceed with setup
 * after setup finalization, the user will be sent to `InboxViewController` with a list of emails
 */
class SetupBackupsViewController { ...
```

Reply:
Yes the flows same as you described. 
Renamed to `SetupManuallyEnterPassPhraseViewController`